### PR TITLE
reorganize reference doc for distance metrics

### PR DESCRIPTION
**Pull request by arnej27959**
@kkraune please review
@jobergum @geirst please take a look also

### CODE REVIEW

**Review by kkraune — APPROVED** [review]
Generally I find this well written and understandable - thanks!  :-)

**Comment by jobergum** [comment]
This is great. I agree that we dont' need to prefix with Notes, this also allow linking directly to e.g `angular`, while the top level mentions all the distance metrics! I love it, and 100% improvement over the previous version.

File: en/reference/schema-reference.html [hunk 1]
```diff
@@ -2675,142 +2675,157 @@ <h2 id="sorting">sorting</h2>
...
-    for your tensor, with the usual encoding from bit pattern to numerical value, for example:
-    <ul>
+    <td>Only useful for binary data, see <a href="#distance-metric-hamming-notes">note</a> below.</td>
+    <td><code>d = sum(popcount(xi XOR yi))</code></td>
+    <td><code>1.0 / (1.0 + d)</code></td>
+  </tr>
+</tbody>
+</table>
+
+<h3 id="distance-metric-angular-notes">Notes: angular</h3>
```
[the remark this answers is not in the document]
Comment (jobergum) [reply en/reference/schema-reference.html:2743]: I agree that we can skip the `Notes:` prefix

**Review by geirst — COMMENTED** [review]
Great 👍

**Pull request merged**